packages/iov-bns/src/bnsconnection.ts

```diff
@@ -745,6 +754,43 @@ export class BnsConnection implements AtomicSwapConnection {
     return votes;
   }
 
+  public async getDeposits(query: BnsDepositQuery): Promise<readonly BnsTermDepositNft[]> {
+    let keyPrefix = "deposit:";
```

Comment:
When you write `let keyPrefix: string`, TypeScript will ensure that you set the value in every code branch. No way to accidentally forget it.